docs/Features/XMLFeatures.md

```diff
@@ -144,6 +144,18 @@ This also works for other portions of an XML document, such as attributes and th
 
 ![Selection Range](../images/Features/SelectionRange.gif)
 
+## CodeLens
+
+CodeLens is supported for `.xml` files for a number of different features.
+
+### CodeLens 'Bind to grammar/schema...'
+
+If the current XML document is not bound to an existing grammar/schema, a CodeLens will appear above the root element, which will trigger the [XML Binding Wizard](../BindingWithGrammar.md#the-xml-binding-wizard).
+
+![CodeLens Bind](../images/Features/CodeLensBind.png)
+
+For more information on CodeLens capabilities, see [CodeLens](../CodeLens.md#code-lens).
```

Comment:
It might be interesting to include a header for XML features in `CodeLens.md`, and mention the CodeLens that opens the binding wizard in this section. You can link to the binding wizard page like you did here, so that you don't have to fully explain its functionality again.

Reply:
Should I move the XML Features  CodeLens image to CodeLens.md?

Reply:
Maybe just show it in both files?